DatWeatherDoe/API/Response/WeatherAPIResponse.swift

```diff
@@ -40,6 +40,123 @@ struct WeatherAPIResponse: Decodable {
             case speed
             case degrees = "deg"
         }
+
+        enum Direction {
```

Comment:
Can you extract all the Direction-related stuff into its own file?

Reply:
Sure thing.